cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
I do understand how we mark AIO, I was talking about a subtly different thing, I probably didn't explain it very well above.
As I'm sure you know, a lot of python (using python as an example because I know it better) stuff can sometimes be significantly slower than equivalent operations, even with the same time complexity, for example some of the python solutions I was testing on are about 10 times slower than C and others I've come across previously are almost as fast. Python is just good at certain operations and bad others, and there are also just some ways to write python that are a lot faster than others. We obviously can't capture all of these subtleties in any simple multiplier.
I can code up a really slow python (python tends to be slowest of all the languages we support) solution to all the AIO problems and then a fast one with a bad complexity and work out multipliers from that, but that would probably take quite a while and I'm not convinced that it's worth it, happy to do it if you think it's important to though
Alternatively we can go with whatever exact values we used to use/orac uses (assuming they exist...)